MSAL/src/util/ios/MSALRedirectUriVerifier.m

```diff
@@ -36,8 +36,8 @@ + (MSALRedirectUri *)msalRedirectUriWithCustomUri:(NSString *)customRedirectUri
                                             error:(NSError * __autoreleasing *)error
 {
 #if AD_BROKER
-    // Allow the broker app to use a special redirect URI when acquiring tokens
-    if ([customRedirectUri isEqualToString:MSID_AUTHENTICATOR_REDIRECT_URI])
+    // Allow the broker app to use any non-empty redirect URI when acquiring tokens
+    if (![NSString msidIsStringNilOrBlank:customRedirectUri])
```

Comment:
After the change, does it mean that no redirect uri verification will be done here if the code is executed in broker? Including Authenticator calling broker as well as other apps calling broker.

Reply:
This is only applicable when Authenticator is using MSAL to call broker, and it will be allowed to use any redirectUri. Other apps calling broker pass a different validation and were not allowed to use a special "urn:" redirectUri.

Reply:
you are right, broker call won't use `MSALRedirectUriVerifier` :)